front/tests/012-op-simulation-settings-tab.spec.ts

```diff
@@ -0,0 +1,356 @@
+  const [homePage, operationalStudiesPage, scenarioPage] = [
+    new HomePage(page),
+    new OperationalStudiesPage(page),
+    new ScenarioPage(page),
+  ];
+
+  // Create a new scenario
+  ({ project, study, scenario } = await setupScenario(electricalProfileSet.id));
+
+  // Navigate to home page and retrieve the language setting
+  await homePage.goToHomePage();
+  selectedLanguage = await homePage.getOSRDLanguage();
+
+  // Navigate to the created scenario page
+  await page.goto(
+    `/operational-studies/projects/${project.id}/studies/${study.id}/scenarios/${scenario.id}`
+  );
+  // Ensure the infrastructure is completely loaded
+  await scenarioPage.checkInfraLoaded();
+  // Add a new train and set its properties
+  await operationalStudiesPage.clickOnAddTrainBtn();
+  await scenarioPage.setTrainScheduleName('Train-name-e2e-test');
+  await page.waitForTimeout(500);
+  await operationalStudiesPage.setTrainStartTime('11:22:40');
+  // Select electric rolling stock
+  await operationalStudiesPage.selectRollingStock(freakRollingStockName);
+  // Perform pathfinding
+  await scenarioPage.openTabByDataId('tab-pathfinding');
+  await operationalStudiesPage.performPathfindingByTrigram('WS', 'SES', 'MWS');
+  // Navigate to the Times and Stops tab and fill in required data
+  await scenarioPage.openTabByDataId('tab-timesStops');
+  await scrollContainer(page, '.time-stops-datasheet .dsg-container');
+});
+
+test.describe('Simulation Settings Tab Verification', () => {
+  // Set viewport to avoid scrolling issues and ensure elements are attached to the DOM
+  test.use({ viewport: { width: 1920, height: 1080 } });
+  test('should correctly activate electrical profiles', async ({ page }) => {
+    const [
+      opInputTablePage,
+      opTimetablePage,
+      opOutputTablePage,
+      scenarioPage,
+      opSimulationSettingsPage,
+    ] = [
+      new OperationalStudiesInputTablePage(page),
+      new OperationalStudiesTimetablePage(page),
+      new OperationalStudiesOutputTablePage(page),
+      new ScenarioPage(page),
+      new OperationalStudiesSimulationSettingsPage(page),
+    ];
+    // Project selected language
+    const translations = selectedLanguage === 'English' ? enTranslations : frTranslations;
+    const cell: CellData = {
+      stationName: 'Mid_East_station',
+      header: 'stopTime',
+      value: '124',
+    };
+
+    const translatedHeader = cleanWhitespace(translations[cell.header]);
+
+    await opInputTablePage.fillTableCellByStationAndHeader(
+      cell.stationName,
+      translatedHeader,
+      cell.value,
+      selectedLanguage
+    );
+    // Activate electrical profiles
+    await scenarioPage.openTabByDataId('tab-simulation-settings');
+    await opSimulationSettingsPage.activateElectricalProfile();
+    // Add the train schedule and verify output results
+    await scenarioPage.addTrainSchedule();
+    await scenarioPage.returnSimulationResult();
+    await page.reload();
```

Comment:
why this reload? and the other l.174?

Reply:
It's a workaround until bug #8854 is fixed
I will add a comment

Reply:
Done